robust-websocket.js

```diff
@@ -11,6 +11,8 @@
   }
 })(function(global, navigator) {
 
+  var WebSocket = global.WebSocket
```

Comment:
As per https://github.com/appuri/robust-websocket/pull/24#discussion_r222839117 

> The reason for this change is: WebSocket has been overriden by browser extensions and users want to ensure that RobustWebSocket's implementation doesn't change from under them.

Reply:
> WebSocket has been overriden by browser extensions and users want to ensure that RobustWebSocket's implementation doesn't change from under them.

But this can make it worse. Based on the load order (which is determined by what?) `RobustWebSocket` could close over the extention's implentation, or the native implementation. In the current usage, with lazy invocation, the user's code has more time and control to inject a different implementation.